Add `--ignore-daemonsets` to docs of `kubectl drain` (#38917)
kubernetes/website: content/en/docs/tasks/administer-cluster/safely-drain-node.md

Next, tell Kubernetes to drain the node:

```shell
kubectl drain --ignore-daemonsets <node name>
```

> If there are daemon set managed pods, drain will not proceed without `--ignore-daemonsets`,
> and regardless it will not delete any daemon set managed pods, 
> because those pods would be immediately replaced by the daemon set controller,
> which ignores unschedulable markings.

Reviewer: ```suggestion
If there are DaemonSet managed pods, drain will usually not succeed unless you specify
`--ignore-daemonsets`. The `kubectl drain` subcommand on its own does not actually drain
a node of its DaemonSet pods:
the DaemonSet controller (part of the control plane) immediately replaces missing Pods with
new equivalent Pods. The DaemonSet controller also creates Pods that ignore unschedulable
taints, which allows the new Pods to launch onto a node that you are draining.
```

How about that?

Author: thank you. Your version is easier to understand.

Next, tell Kubernetes to drain the node:

```shell
kubectl drain --ignore-daemonsets <node name>
```

> If there are DaemonSet managed pods, drain will usually not succeed unless you specify

Reviewer 2: IMO, this sounds better and idiomatic without taking away from the meaning. WDYT?
```suggestion
If there are pods managed by a DaemonSet, you will need to specify
```

Author: yes, your wording is easier to read. Thank you.

> `--ignore-daemonsets`. The `kubectl drain` subcommand on its own does not actually drain

Reviewer 2: ```suggestion
`--ignore-daemonsets` with `kubectl` to successfully drain the node. The `kubectl drain` subcommand on its own does not actually drain
```
